docs/x509/reference.rst

```diff
@@ -1695,6 +1695,27 @@ X.509 Extensions
 
         :type: int
 
+.. class:: CRLNumber(crl_number)
+
+    .. versionadded:: 1.2
+
+    The CRL number is a non-critical CRL extension that conveys a
+    monotonically increasing sequence number for a given CRL scope and
+    CRL issuer. This extension allows users to easily determine when a
+    particular CRL supersedes another CRL. :rfc:`5280` requires that this
+    extension be present in conforming CRLs.
```

Comment:
So, it has to be included but should never be marked critical? Ugh, the IETF is teh worst.